bittensor/core/extrinsics/utils.py

```diff
@@ -12,14 +14,7 @@
     from substrateinterface import SubstrateInterface, ExtrinsicReceipt
     from scalecodec.types import GenericExtrinsic
 
-
-class _ThreadingTimeoutException(Exception):
-    """
-    Exception raised for timeout. Different from TimeoutException because this also triggers
-    a websocket failure. This exception should only be used with `threading` timer..
-    """
-
-    pass
+EXTRINSIC_SUBMISSION_TIMEOUT = int(os.getenv("EXTRINSIC_SUBMISSION_TIMEOUT", 200))
```

Comment:
```suggestion
_EXTRINSIC_SUBMISSION_TIMEOUT_default = 200
try:
    EXTRINSIC_SUBMISSION_TIMEOUT = int(os.getenv("EXTRINSIC_SUBMISSION_TIMEOUT", _EXTRINSIC_SUBMISSION_TIMEOUT_default))
except ValueError:
    EXTRINSIC_SUBMISSION_TIMEOUT = _EXTRINSIC_SUBMISSION_TIMEOUT_default
```
Need to handle the int parse here or we crash on import for bad env var